groups.go

```diff
@@ -345,6 +345,40 @@ func (s *GroupsService) TransferGroup(gid interface{}, pid interface{}, options
 	return g, resp, err
 }
 
+// TransferSubGroupOptions represents the available TransferSubGroup() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/groups.html#transfer-a-group-to-a-new-parent-group--turn-a-subgroup-to-a-top-level-group
+type TransferSubGroupOptions struct {
+	ParentID *int `url:"group_id,omitempty" json:"group_id,omitempty"`
```

Comment:
Let's make sure we follow the Gitlab naming:
```suggestion
	GroupID *int `url:"group_id,omitempty" json:"group_id,omitempty"`
```

Reply:
I had expected this feedback. :blush: 
GroupID is confusing for me, because it is not clear which group is meant. But you are right, you have to follow the Gitlab naming.